README.md

```diff
@@ -25,16 +25,13 @@ Or install it yourself as:
 
 Simply initialize a client with a host and your token. You can optionally generate secure URLs.
 
-Now, if you have the URL ready to go, you can call `sign_path` to get the Imgix URL back. If you would like to manipulate the path parameters you can call `path` with the resource path to get an Imgix::Path object back.
+Call `Imgix::Client#path` with the resource path to get an `Imgix::Path` object back. You can then manipulate the path parameters, and call `Imgix#Path#to_url` when you're done.
```

Comment:
Rather than say "when you're done," we should be more explicit. "... and call `Imgix::Path#to_url` to generate a `String` of the built URL."